spec/matchers_spec.lua

```diff
@@ -288,12 +288,12 @@ describe("Test Matchers", function()
   it("Checks returned_with() argument matching for spies", function()
     local s = spy.new(function() return "foo" end)
     s()
-    assert.spy(s).was.returned_with(match._)
-    assert.spy(s).was.returned_with(match.is_string())
-    assert.spy(s).was.returned_with(match.is_not_number())
-    assert.spy(s).was.returned_with(match.is_not_table())
-    assert.spy(s).was_not.returned_with(match.is_number())
-    assert.spy(s).was_not.returned_with(match.is_table())
+    assert.spy(s).returned_with(match._)
+    assert.spy(s).returned_with(match.is_string())
+    assert.spy(s).returned_with(match.is_not_number())
+    assert.spy(s).returned_with(match.is_not_table())
+    assert.spy(s).has_not.returned_with(match.is_number())
+    assert.spy(s).has_not.returned_with(match.is_table())
```

Comment:
there are many of these types changes. Are they related? seems not. Can you please remove those?

Reply:
Gone.